src/command_modules/azure-cli-security/azure/cli/command_modules/security/commands.py

```diff
@@ -0,0 +1,176 @@
+    )
+
+    security_locations_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.security.operations.locations_operations#LocationsOperations.{}',
+        client_factory=cf_security_locations,
+        operation_group='security_locations'
+    )
+
+    security_pricings_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.security.operations.pricings_operations#PricingsOperations.{}',
+        client_factory=cf_security_pricings,
+        operation_group='security_pricings'
+    )
+
+    security_topology_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.security.operations.topology_operations#TopologyOperations.{}',
+        client_factory=cf_security_topology,
+        operation_group='security_topology'
+    )
+
+    security_workspace_settings_sdk = CliCommandType(
+        operations_tmpl='azure.mgmt.security.operations.workspace_settings_operations#WorkspaceSettingsOperations.{}',
+        client_factory=cf_security_workspace_settings,
+        operation_group='security_workspace_settings'
+    )
+
+    with self.command_group('security task',
+                            security_tasks_sdk,
+                            client_factory=cf_security_tasks) as g:
+        g.custom_command('list', 'list_security_tasks')
+        g.custom_command('show', 'get_security_task')
+
+    with self.command_group('security alert',
+                            security_alerts_sdk,
+                            client_factory=cf_security_alerts) as g:
+        g.custom_command('list', 'list_security_alerts')
+        g.custom_command('show', 'get_security_alert')
+        g.custom_command('dismiss', 'dismiss_security_alert')
+        g.custom_command('activate', 'activate_security_alert')
+
+    with self.command_group('security setting',
+                            security_settings_sdk,
+                            client_factory=cf_security_settings) as g:
+        g.custom_command('list', 'list_security_settings')
+        g.custom_command('show', 'get_security_setting')
+
+    with self.command_group('security contact',
+                            security_contacts_sdk,
+                            client_factory=cf_security_contacts) as g:
+        g.custom_command('list', 'list_security_contacts')
+        g.custom_command('show', 'get_security_contact')
+        g.custom_command('create', 'create_security_contact')
+        g.custom_command('delete', 'delete_security_contact')
+
+    with self.command_group('security auto-provisioning-setting',
+                            security_auto_provisioning_settings_sdk,
+                            client_factory=cf_security_auto_provisioning_settings) as g:
+        g.custom_command('list', 'list_security_auto_provisioning_settings')
+        g.custom_command('show', 'get_security_auto_provisioning_setting')
+        g.custom_command('turn-on', 'turn_on_security_auto_provisioning_setting')
+        g.custom_command('turn-off', 'turn_off_security_auto_provisioning_setting')
```

Comment:
`turn-off` and `turn-on` should be incorporated in an update command.

Reply:
updated